jobbergate-api/jobbergate_api/apps/applications/routers.py

```diff
@@ -94,12 +96,11 @@ async def applications_list(
     """
     armada_claims = ArmadaClaims.from_token_payload(token_payload)
     query = applications_table.select()
-    if not all:
+    if user:
         query = query.where(applications_table.c.application_owner_email == armada_claims.user_email)
-    query = query.limit(p.limit).offset(p.skip)
-    raw_applications = await database.fetch_all(query)
-    applications = [Application.parse_obj(x) for x in raw_applications]
-    return applications
+    if all is None:
+        query = query.where(not_(applications_table.c.identifier.is_(None)))
```

Comment:
No, because the `user` param and the `all` param are independent of each other. Both can be used at the same time, so we need to check them separately.